src/steamship/client/client.py

```diff
@@ -104,6 +106,46 @@ def embed_and_search(
             space=space,
         )
 
+    def use(
+        self, app_handle: str, handle: str = None, config: Dict[str, Any] = None
+    ) -> AppInstance:
+        """Creates or loads an instance named `handle` of App named `app_handle`."""
+        instance = AppInstance.create(
+            self,
+            app_handle=app_handle,
+            handle=handle,
+            config=config,
+            upsert=True,  # TODO(ted): Rename `upsert` globally to something semantically appropriate
```

Comment:
Up for debate: Should `upsert` be True or False by default? 

Always creating Apps might pollute the workspace of our customers. 

Side note: Lately I've been using the term `workspace` to remind me what a `space` means. To me it works well as a mental model. A workspace is a place where I can think around with apps, plugins, and data. If I want to start from scratch, I just walk away from my workspace and go to a new one. If not I have to clean my workspace. 

I'm also dreaming that the client would be the mediator for workspaces as a user. This would like as follows: 

```
client = Steamship() 
File.create(...) # Create a file in my default workspace
client.workspace("new-workspace-for-lazy-enias") 
File.create(...) # This creates a file in my new workspace  
```

Reply:
Yeah; I think we're due for a round of naming experimentation before we launch.

Similarly, I've started really liking `package` instead of `app` in the way I think/talk about things.

Regarding `client.workspace(...)` I agree that some notion of a global client might be really cool. Taking that idea further, it would support:

```
File.create(...)
```

Without **any** `client = Steamship()` if we wanted the "what to do if `client=None` to mean `client=get_global_client()` which itself noticed the lack of one and created it...